style.css

```diff
@@ -302,3 +302,69 @@ button:disabled {
   border-radius: 0;
   color: #000;
 }
+
+.contacts {
+  background-image: url("./images/contactshapes.png");
+  display: flex;
+  background-color: #396df2;
+  flex-direction: column;
+  justify-content: space-around;
+  width: 100%;
+  background-repeat: no-repeat;
+  background-position: right;
+  background-size: 90% 100%;
+  border-top-left-radius: 100px;
+}
+
+.contact-section {
+  display: flex;
+  flex-direction: column;
+  width: 90%;
+  gap: 10px;
+  margin: 0 auto 50px;
+  background-repeat: no-repeat;
+}
+
+.contact-intro {
+  margin-top: 20px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.contact-me {
+  font-family: "poppins", sans-serif;
+  margin-block-end: 0;
+  font-size: 40px;
+  font-weight: 900;
+  color: #fff;
+}
+
+.paragraph {
+  width: 380px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 25px;
```

Comment:
- [ ] Please use the correct properties for fonts such as size line-height and weight, as it is mentioned in the figma template, please remember we have to stick to the design as much as possible (e.g., font, colors, images, text, margins)
from figma :point_down: 
```
font-size: 20px;
font-weight: 400;
line-height: 28px;
text-align: center;

```

- [ ] Please also try to match the figma design for your contact section's paragraph as well as for input fields, see your design :point_down: 

![image](https://user-images.githubusercontent.com/97521557/212973989-ed8a1189-e17a-451a-86c7-0e1a59f70e4c.png)

see figma design :point_down: 
![image](https://user-images.githubusercontent.com/97521557/212974133-714753a4-bf93-4af0-9a06-afa8a10e8643.png)

Notice the padding for input fields, and how the text is centered on the page with some right and left padding